boards/arc/emsdp/doc/index.rst

```diff
@@ -44,6 +44,9 @@ The following table shows the hardware features supported for different core con
 | Secure    | N   | N   | N    | N    | Y        | N    | N     |
 +-----------+-----+-----+------+------+----------+------+-------+
 
+Support two 32 MByte Quad-SPI Flash memory, one is for configuration and operation, the other
```

Comment:
Wording is a bit strange. Also from EM SDP User Guide (see https://github.com/foss-for-synopsys-dwc-arc-processors/ARC-Development-Systems-Forum/wiki/docs/ARC_EM_SDP_User_Guide.pdf)  I may conclude that there're 2 flash IC's connected via SPI bus each of which is 16 MiB, not 32 (i.e. it's 16+16 which gives 32 Mib in total). Moreover one is meant to be used for on-board FPGA image storage, while another one is dedicated for all user needs.

That said, I'd phrased it like that:
> The board contains two 16 MiB flash IC's connected to the ARC core via SPI bus. One contains FPGA image and another one is dedicated to user needs.

Also it would be good to add more details on how to address a particular IC so that your fellow user unexpectedly don't overwrite FPGA image effectively bricking the board.

Reply:
updated in new doc. Not both of two flashs are connected via spi bus.